oop-essentials/ide-features/README.md

```diff
@@ -157,14 +157,14 @@ Auto-completion helps save time by not requiring the developer to type out the n
 | IDE       | Shortcut  |
 | --------- | --------- |
 | JetBrains | just start typing or explicitly request via Ctrl+Space |
-| VSCode    | TODO |
+| VSCode    | just start typing |
 
 **Auto-completion of identifiers in the entire project**, including external dependencies:
 
 | IDE       | Shortcut  |
 | --------- | --------- |
 | JetBrains | Ctrl+Space+Space |
-| VSCode    | TODO |
+| VSCode    | just start typing |
```

Comment:
Are you sure this works? In this case, we are trying to find an identifier which has not yet been imported in the current model and which could be anywhere (in the same project or its dependencies)!

Reply:
I just verified that it works for in-project identifiers but not for dependencies.
Any way to find and auto-import, for example, StandardScaler from sklearn?

Reply:
Mmmh I do not know of anything working for dependencies...

Reply:
ah wait, you need to activate the python>Analysis: Auto Import Completions in settings